pkgs/os-specific/linux/zfs/stable.nix

```diff
@@ -14,10 +14,11 @@ callPackage ./generic.nix args {
   # check the release notes for compatible kernels
   kernelCompatible =
     if stdenv'.isx86_64 || removeLinuxDRM
-    then kernel.kernelOlder "6.4"
+    then kernel.kernelOlder "6.6"
     else kernel.kernelOlder "6.2";
-  latestCompatibleLinuxPackages = linuxKernel.packages.linux_6_1;
+  latestCompatibleLinuxPackages = linuxKernel.packages.linux_6_5;
```

Comment:
```suggestion
  latestCompatibleLinuxPackages = if stdenv'.isx86_64 || removeLinuxDRM
    then linuxKernel.packages.linux_6_4
    else linuxKernel.packages.linux_6_1;
```

Reply:
I understand this now that I looked up the module option for removeLinuxDRM. Interestingly #236459 didn't have this logic. Should we maybe just leave it in here moving forward even if 6.5 gets dropped before the next zfs version?

Reply:
Yes.